### PR TITLE
test(e2e): use e2e-mayastor test files

### PR DESCRIPTION
**Pull request by chriswldenyer**
Mayastor e2e tests should get their test files from the mayastor-e2e repo
Removed the e2e test files from Mayastor.
Jenkins obtains the e2e test files from mayastor-e2e

### CODE REVIEW

**Comment by chriswldenyer** [comment]
bors try

**Comment by chriswldenyer** [comment]
bors try

**Comment by chriswldenyer** [comment]
bors try

**Comment by chriswldenyer** [comment]
bors try

**Comment by chriswldenyer** [comment]
bors try

**Comment by chriswldenyer** [comment]
bors try

**Comment by chriswldenyer** [comment]
bors try

**Comment by chriswldenyer** [comment]
bors try

File: Jenkinsfile [hunk 1]
```diff
@@ -20,6 +20,7 @@ xray_send_report = (env.BRANCH_NAME == 'trying') ? false : true
 
 // if e2e run does not build its own images, which tag to use when pulling
 e2e_continuous_image_tag='v0.8.0'
+e2e_reports_dir='artifacts/reports/'
```
Comment (blaisedias) [inline Jenkinsfile:23]: Shouldn't this be 'mayastor-e2e/artifacts/reports/'?

**Review by GlennBullingham — APPROVED** [review]
LGTM

**Review by blaisedias — APPROVED** [review]
LGTM

**Comment by chriswldenyer** [comment]
bors merge

**Pull request merged**